app/assets/stylesheets/fortitude/components/_layout.scss

```diff
@@ -4,7 +4,7 @@ title: Layout
 name: layout
 category: Structure
 ---
-Used for content that needs a grid default gutter width is `2rem` or `20px`
+Used to create a responsive horizontal grid system with gutter areas between columns. The default gutter width is `2rem` or `20px`.
```

Comment:
Probably better to refer to it in terms of base line height, but non-critical for now.